packages/styles/forms/_inputs.scss

```diff
@@ -130,6 +130,20 @@
   }
 }
 
+.k-input-wrapper textarea.form-control {
+  padding: 17px 0 0 22px !important;
+
+  &:focus {
+    box-shadow: inset 0 0 0 2px var(--KInputFocus, var(--blue-400)) !important;
+  }
+}
+
+textarea.form-control {
+  font-family: 'Maison Neue';
```

Comment:
should be using `font(sans)` if you need a font-family, but I think this should be the default.